cidc_api/models.py

```diff
@@ -74,30 +73,35 @@ class ExtraDataTypes(EnumBaseClass):
 ]
 
 
+class FKResourceConfig(ResourceConfig):
+    def _get_column_fields(self):
+        """
+        Eve-SQLAlchemy's built-in ResourceConfig generator doesn't include foreign keys
+        without relationships in the schema config, but we want to allow this, so
+        we override this method: https://github.com/pyeve/eve-sqlalchemy/blob/22bdd96fe5b461fbdfe4a4dfdc394569f7e9b8b4/eve_sqlalchemy/config/resourceconfig.py#L157
```

Comment:
we override this method
```suggestion
        we override this method: https://github.com/pyeve/eve-sqlalchemy/blob/22bdd96fe5b461fbdfe4a4dfdc394569f7e9b8b4/eve_sqlalchemy/config/resourceconfig.py#L157
        by removing `len(f.expression.foreign_keys) == 0)` filter from it
```